.github/workflows/deploy-book.yml

```diff
@@ -24,6 +24,12 @@ env:
   # By default, preprocess no branches ('').
   # Preprocessing no branches can also be set by ' '.
   BRANCHES_TO_PREPROCESS: ${{ vars.BRANCHES_TO_PREPROCESS }}
+  # If to copy the primary page to root
+  COPY: ${{ vars.COPY != '' && vars.COPY || 'false' }}
+  # Space-separated list of branch names
+  # Note: does not support '*'
+  BRANCHES_ARCHIVED: ${{ vars.BRANCHES_ARCHIVED }}
+
```

Comment:
add default value

Reply:
The default value as of now is empty (empty string, so empty list). I think it should stay this way, no?